.github/workflows/gh-pages.yml

```diff
@@ -107,6 +111,17 @@ jobs:
             fi
           done
 
+      - name: Build | JS API Reference
+        run: |
+          cp -r js/* src/wasm
+          cd src/wasm
+          mkdir doc
+          node api_doc.js
+          jsdoc doc/readable_api.js -d doc/out
+          cd ../..
+          mkdir -p build/js
+          cp -r src/wasm/doc/out build/js/main
```

Comment:
Shouldn't this be `build/docs/js/main`?

Reply:
I don't think so, the documentation for OCaml bindings is copied to `build/ocaml/main`.

Reply:
Okay, the local build seems to diverge from the CI one. Let's investigate later.